### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by ifvictr**
**2** grammar fixes made.

### CODE REVIEW

File: src/HereAuth/HereAuth.php [hunk 1]
```diff
@@ -221,7 +221,7 @@ public function onDisable(){
 	public function startUser(Player $player){
 		$this->database->loadFor($player->getName(), $player->getId());
 		if($player->spawned){
-			$player->sendMessage("[HereAuth] Your account data are being loaded. Please wait patiently; it shouldn't take long.");
+			$player->sendMessage("[HereAuth] Your account data is being loaded. Please wait patiently; it shouldn't take long.");
```
Comment (PEMapModder) [inline src/HereAuth/HereAuth.php:224]: data is the plural of datum.

Comment (ifvictr) [inline src/HereAuth/HereAuth.php:224]: You are referring to all the data as one thing when you use `data`, therefore using `is` would make more sense, unless you want to use datum, which in that case, an `are` would be adequate. I'll leave this up to you to decide on.

**Comment by SOF3** [comment]
On the day he died.

File: src/HereAuth/HereAuth.php (hunk 1 above)
Comment (SOF3) [inline src/HereAuth/HereAuth.php:224]: @ifvictr you say "Your pants **are** on fire", not "Your pants **is** on fire", even though you are just referring to all the pants in your pair of pants.

Comment (ifvictr) [inline src/HereAuth/HereAuth.php:224]: @SOF3 Yes, you're strictly following the rules of grammar. But to majority of English speakers nowadays, it wouldn't make sense (due to datum rarely being used, data now refers to both the singular and plural form of data). I believe that whatever is used more often should be used, and which, in this case, is "data is". However, this is up to @PEMapModder , to decide. https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular

**Pull request merged**